app/js/app/directives/react_classes/MetaData.js

```diff
@@ -556,8 +557,9 @@ define(["react", "jquery"], function(React,$) {
 								<div className="small-10 columns">{tagsComponent}</div>
 							</div>
 							<div className="row">
+								{idInvalid && <div className="columns text-right">Please alter this ID, as it does not match our recommended style</div>}
```

Comment:
I think I'd prefer "required" style, since some of these matching IDs cannot be used by ElasticSearch at all and are just unusable rather than not "recommended".